org.kde.kfloppy.json

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "org.kde.kfloppy",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kfloppy",
+    "rename-icon": "kfloppy",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host"
+    ],
+
+    "modules": [
+        {
+            "name": "kfloppy",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { "type": "git",
+                  "commit": "c98dbb8c326b380bac0b23233c9015716815923f",
+                  "url": "https://invent.kde.org/kde/kfloppy.git" },
+                { "type": "patch",
+                  "path": "appdata.patch"
+                }
```

Comment:
did you test this application? According to the application description, it uses external commands, but I doubt they are available in the flatpak sandbox.